F test_demo.py::TestClass::test_func
 self = <test_demo.TestClass object at 0x10a428160>
 
     def test_func(self):
 >       assert self.func(3) == 5
 E       assert 4 == 5
 E         -4
 E         +5
 
 test_demo.py:10: AssertionError
